api/src/routing.js

```diff
@@ -111,6 +112,18 @@ app.postJsonOrCsv = (path, callback) => handleJsonOrCsvRequest('post', path, cal
 app.postJson = (path, callback) => handleJsonRequest('post', path, callback);
 app.putJson = (path, callback) => handleJsonRequest('put', path, callback);
 
+app.use(prometheusMiddleware({
+  metricsPath: '/api/v1/express-metrics',
```

Comment:
@jkuester There is an option for a [metricsPrefix](https://github.com/PayU/prometheus-api-metrics#options). Should I set it to `cht_`? `cht_api_`? Do you care?

Reply:
My vote is for `cht_api` as the best way to avoid collisions with names and keep everything straight.